.github/workflows/blank.yml

```diff
@@ -0,0 +1,27 @@
+name: Github Actions CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Docker version
+      run: docker version
+    - name: Debug if needed
+      run: |
+        [[ "$DEBUG" ]] && env
+      env:
+        DEBUG: ${{secrets.DEBUG}}
+    - name: Build image
+      run: |
+        export GIT_BRANCH="${GITHUB_REF/refs\/heads\//}"
```

Comment:
I'm not sure what this GIT_BRANCH  for, but will it work for tags?

Reply:
https://github.com/mattn/goveralls/blob/master/gitinfo.go#L78 this case. I check about using tag